models/istio_validation.go

```diff
@@ -147,6 +150,14 @@ var checkDescriptors = map[string]IstioCheck{
 		Message:  "Mesh-wide Destination Rule enabling mTLS is missing",
 		Severity: ErrorSeverity,
 	},
+	"servicerole.invalid.services": {
+		Message:  "Unable to find all the defined services",
+		Severity: ErrorSeverity,
+	},
+	"servicerole.invalid.namespace": {
+		Message:  "ServiceRole can only point to current namespace",
+		Severity: ErrorSeverity,
+	},
```

Comment:
There is missing one servicerolebinding validation definition in here.
I guess that it makes the list show the following:
![istio-validation-missing-validation-descriptor](https://user-images.githubusercontent.com/613814/54542295-d7281e80-499b-11e9-82c4-0b7632c62ce4.png)
(the last item in the list should show an error)

And, as @hhovsepy mentioned, in the yaml page, we see this:
![servicebinding-validation-error](https://user-images.githubusercontent.com/613814/54542436-0b9bda80-499c-11e9-83ad-69561d600998.png)

Reply:
yes the notification message is missing here